aci/resource_aci_psuinstpol.go

```diff
@@ -0,0 +1,203 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAciPowerSupplyRedundancyPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciPowerSupplyRedundancyPolicyCreate,
+		UpdateContext: resourceAciPowerSupplyRedundancyPolicyUpdate,
+		ReadContext:   resourceAciPowerSupplyRedundancyPolicyRead,
+		DeleteContext: resourceAciPowerSupplyRedundancyPolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciPowerSupplyRedundancyPolicyImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+
+			"administrative_state": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"comb",
+					"insrc-rdn",
+					"n-rdn",
+					"not-supp",
+					"ps-rdn",
+					"rdn",
+					"sinin-rdn",
+					"unknown",
+				}, false),
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		})),
+	}
+}
+
+func getRemotePowerSupplyRedundancyPolicy(client *client.Client, dn string) (*models.PsuInstPol, error) {
+	psuInstPolCont, err := client.Get(dn)
+	if err != nil {
+		return nil, err
+	}
+	psuInstPol := models.PsuInstPolFromContainer(psuInstPolCont)
+	if psuInstPol.DistinguishedName == "" {
+		return nil, fmt.Errorf("PowerSupplyRedundancyPolicy %s not found", dn)
```

Comment:
Please change:

"PowerSupplyRedundancyPolicy %s not found" -> "Power Supply Redundancy Policy %s not found"

Reply:
@shrsr Every single piece of feedback is valuable, thanks!